FPM.ftd

```diff
@@ -99,6 +99,7 @@ implements: fifthtry.github.io/theme
 - Journal: /journal/
 - Contribution Guide: /dev/contribute/
 - How Versioning Works: /dev/versioning/
+- How to Use Favicon: /dev/how-to-use-favicon/
```

Comment:
This is not a dev topic. It should go in the author manual section.

Reply:
The URL of the doc can be fpm.dev/favicon/ (so create a top level favicon.ftd file).

Reply:
Alright

Reply:
The URL is still not /favicon/. Please rename the file to favicon.ftd.

Reply:
Done